### ISSUE TITLE
英语邮件常用

### ISSUE THREAD

**Issue by CodeJuan** [CodeJuan/blog#84]
```
工作中，少不了各种各样的工作邮件，特别是在外企更是日常必备了。商务邮件用英语书写已经很广分了，下面给大家介绍一些英文电邮里的高频句子，让你在职场中游刃有余。

1. I am writing to confirm/enquire/inform you...

我发邮件是想找你确认/询问/想通知你 有关…

2. I am writing to follow up on our earlier decision on the marketing campaign in Q2.

我写邮件来是为了跟进我们之前对第二季度营销活动的决定。

3. With reference to our telephone conversation today...

关于我们今天在电话中的谈话…

4. In my previous e-mail on October 5...

在之前10月5日所写的邮件中提到…

5. As I mentioned earlier about...

在先前我所提到的关于…

6. As indicated in my previous e-mail...

如我在之前邮件中所提到的…

7. As we discussed on the phone...

如我们上次在电话中所说的…

8. from our decision at the previous meeting...

如我们在上次会议中所决定的…

9. as you requested...

根据贵方要求…

10. In reply to your e-mail dated April 1，we decided...

回复贵方4月1日的邮件，我方决定…

11. This is in response to your e-mail today.

这是对您今早发来的邮件的回复。

12. As mentioned before, we deem this product has strong unique selling points in china.

如先前所述，我们认为这个产品在中国有强有力且独一无二的销售点。

13. As a follow-up to our phone conversation yesterday, I wanted to get back to you about the pending issues of our agreement.

追踪我们昨天在电话中所谈，我想答复你我们合约的一些待解决的议题。

14. I received your voice message regarding the subject. I'm wondering if you can elaborate i.e. provide more details.

我收到了你关于这个主题的留言。我在想您你是否可以再详尽的说明一下，也就是再提供多一点相关细节。

15. Please be advised/informed that...

请被告知...



16. Please note that...

请注意...

17. We would like to inform you that...

我们想要通知你...

18. I am convinced that...

我确信...

19. We agree with you on...

我们同意你在...的看法。

20. With effect from 4 Oct., 2008...

从2008年10月4日开始生效...

21. We will have a meeting scheduled as noted below...

我们将举行一个会议，会议时间表如下。

22. Be assured that individual statistics are not disclosed and this is for internal use only.

请确保个人信息不会外泄且只供内部使用。

23. I am delighted to tell you that...

我很高兴地告诉你...

24. We are pleased to learn that...

我们很高兴得知...

25. We wish to notify you that...

我们希望通知你...

26. Congratulation on your...

恭喜您...

27. I am fine with the proposal.

我对这份提案没异议。

28. I am pleased to inform you that you have been accepted to join the workshop scheduled for 22-24 Nov,2008.

我十分高兴地通知你，我们已经同意您参加于2008年11月22-24日举行的研讨会。

29. We are sorry to inform you that...

我们很抱歉地通知您...

30. I'm afraid I have some bad news. 

恐怕我今天要宣布一些坏消息。

31. There are a number of issues with our new system.

我们的新系统有些问题。

32. Due to circumstances beyond our control...

由于情况超出我们的控制...

33. I don't feel too optimistic about...

我对...不太乐观

34. It would be difficult for us to accept...

我们很难接受...

35. Unfortunately I have to say that, since receiving your enquiries on the subject, our view has not changed.

我不得不这么说，自从收到你关于这个主题的询问，我们的看法都没有改变。



36. We would be grateful if you could...

如果你可以...我们会很感激的。

37. I could appreciate it if you could...

如果你可以...我会很感激。

38. Would you please send us…?

你能否寄给我们…?

39. We need your help.

我们需要你的帮助。

40. We seek your assistance to cascade/reply this message to your staff.

我们希望您能将此信息传达给你们的员工，非常感谢。

41. We look forward to your clarification.

我们期待你的澄清。

42. Your prompt attention to this matter will be appreciated.

您若能立即关注此事，我们将非常感激。

43. I would really appreciate meeting up if you can spare the time. Please let me know what suits you best.

如果您能从百忙中抽出时间，我希望能与您见面，请让我知道您最适合的时间。

44. Please give us your preliminary thoughts about this.

请让我知道您对这件事情初步的想法。

45. Would you please reply to this e-mail if you plan to attend?

请您回信如果您计划参加?

46. Please advise if you agree with this approach.

请告知我们你是否同意这个方法。

47. Could you please let me know the status of this project?

请通知我这个计划的进度?

48. If possible, I hope to receive a copy of your proposal when it is finished.

如果可能，当你完成提案，我希望你能发一份复本给我。

49. I would appreciate it very much if you would send me your reply by next Monday.

如果能在下周一前收到您的答复，我将非常感激。

50. Hope this is OK with you. If not, let me know by e-mail ASAP.

希望您满意，如果不满意，请发邮件尽快通知我。
```

```
工作中，少不了各种各样的工作邮件，特别是在外企更是日常必备了。商务邮件用英语书写已经很广分了，下面给大家介绍一些英文电邮里的高频句子，让你在职场中游刃有余。

51. Could you please send me your replies to the above questions by the end of June?

请您在6月份前答复我上述问题好吗?

52. May I have your reply by April 1, if possible?

如果可能，我可否在4月1日前收到您的答复?

53. If you wish, we would be happy to...

如果你有需要，我们很乐意...

54. Please let me know if there's anything I can do to help.

如果我有什么可以帮得上忙的, 请告诉我。

55.If there's anything else I can do for you on/regarding this matter, please feel free to contact me at any time.

对于这件事，如果还有什么我能帮得上忙的地方，请随时与我联络。

56.If you want additional recommendations on this, please let us know and we can try to see if this is possible.

如果关于此事你需要额外的建议，请让我们知道，我们会试着看看这是否可行。

57. I'm just writing to remind you of...

我只是写信来提醒您...

58. May we remind you that...?

我们想要提醒您...

59. I am enclosing...

我附上...

60. Please find enclosed...

请查阅附件...

61. Attached hereto...

附件是关于...

62. Attached please find the most up-to-date information on/regarding/concerning…

附上关于某某的最新资料…

63. Attached please find the draft product plan for your review and comment.

附上产品计划书的草稿，请审查及评价。

64. If you have any further questions, please feel free to contact me.

如果你有任何问题，请随时与我联系。

65. I hope my clarification has been helpful.

希望我的说明对你有所帮助。



66. Please feel free to call me at any time, I will continually provide full support.

请随时与我联系，我将持续地提供全程支持。

67. Please let me know if this is suitable.

请让我知道这是否恰当。

68. Looking forward to seeing you soon.

很期待能尽快见到你。

69. We look forward to hearing from you soon.

我们很期待能得到您的回复。

70. Hope this is clear and we are happy to discuss this further if necessary.

希望上述说明很清楚，如有必要，我们很乐意再进一步讨论。

71. I look forward to receiving your reply soon.

我期待很快能收到你的回复。

72. Looking forward to receiving your comments in due course.

期待在预期的时间收到你的反馈。

73. I'll keep you posted.

我会与你保持联系。

74. Please keep me informed on the matter.

请随时让我知道这件事的发展。

75. For any comments/suggestions, please contact Nadia at 2552-7482.

有任何评价或建议，请拨打2552-7482联络Nadia。

76. I would like to apologize for...

我想就...道歉...

77. I apologize for the delay in...

对于...的耽搁，我深感抱歉。

78. We are sorry for any inconvenience caused.

对于产生的任何不便，我们感到抱歉。

79. I am sorry for any inconvenience this has caused you.

对于造成的不便，我深感抱歉。

80. I'm sorry about last time.

上次的事我很抱歉。

81. We apologize for not replying you earlier.

对于未能早一点回信给你，我们感到抱歉。

82. I'm really sorry about this.

这件事，我真的很抱歉。

83. Sorry, I'm late in replying to your e-mail dated Monday, April 1.

抱歉，太迟回您在4月1日(星期一)发给我的邮件。

84. We apologize for the delay and hope that it doesn't inconvenience you too much.

我们为耽搁道歉，希望这没有给您带来太多不便。

85.Hoping that this will not cause you too much trouble.

希望不会为您造成太多麻烦。



86.Sorry if my voice message is not clear enough.

如果我的电话留言不够清楚,我深感抱歉。

87. Thank you for your help.

谢谢你的帮助。

88. I appreciate very much that you...

我非常感激你...

89. I truly appreciate it.

我真的很感激。

90. Thank you for your participation.

谢谢你的参与。

91. Thank you so much for inviting me.

非常感谢您的邀请。

92. Congratulations to all of you and thanks for your efforts.

恭喜各位并十分感谢各位所做的努力。

93. Your understanding and cooperation is greatly/highly appreciated.

很感激您的理解及合作。

94. Your prompt response will be most appreciated.

很感激您快速的答复。

95. Once again, thank you all for your commitment and support.

再一次感谢大家的承诺及支持。

96. Thanks for your input/clarification/message.

谢谢你的投入/澄清/信息。

97. Any comments will be much appreciated.

对于您提出的任何建议，我将深表感谢。

98. Thank you very much for everything you've done for me.

谢谢你为我所做的一切。

99. I would appreciate your kindest understanding with/regarding this matter.

我很感激你对这件事情的理解。

100. Please convey my thanks to all the staff involved, they certainly did an excellent job.

请向有关的同仁传达我的谢意，他们真的干得很好。
```

**Comment by TimelordRi** [comment]
Because gradient accumulation will cause a certain degree of performance degradation, it is recommended to use multiple GPU parallel training to increase the batch size.

As for the hyperparameters mentioned in supplimentray, it is because the optimized parameters forgot to update later.

Finally, the file ’utils_sample.py‘ is uploaded now. Thanks for your issues.
